### PR TITLE
Prettify unpack assessment item output

### LINKED ISSUE #4719: Prettify unpack_assessment_zip output
On `develop`:

With the Windows installer, we now call `unpack_assessment_zip` as part of the installation process. It's output is displayed on a console, but it's really minimal (`Unpacking...`) and can take a long time. Some users will undoubtedly think that it is hung, and inadvertently cancel it, resulting in a failed installation. So let's give it better output -- some kind of progress bar. Something that will quell the rising panic in our user's throats.

### PR DESCRIPTION
**Pull request by MCGallaspy**
Fixes #4719. See commit messages.

### CODE REVIEW

**Comment by benjaoming** [comment]
Cool! :+1:

**Comment by MCGallaspy** [comment]
Any hope of this getting merged? Or should I close it?

**Comment by aronasorman** [comment]
Since we decided to keep `unpack_assessment_zip` around, it makes sense to merge this in. I can review this tomorrow, can you resolve the merge conflicts?

**Comment by MCGallaspy** [comment]
Merge conflicts resolved, but I'll rebase again to see if it clears up test failures.

File: kalite/contentload/management/commands/unpack_assessment_zip.py [hunk 1]
```diff
@@ -61,9 +63,19 @@ def handle(self, *args, **kwargs):
...
+            zf = zipfile.ZipFile(f, "r")
+            unpack_zipfile_to_content_folder(zf)
+
+        unpack_thread = threading.Thread(target=unpack)
+        unpack_thread.daemon = True
+        unpack_thread.start()
         logging.info("Unpacking...")
-        zf = zipfile.ZipFile(f, "r")
-        unpack_zipfile_to_content_folder(zf)
+        while unpack_thread.is_alive():
```
Comment (aronasorman) [inline kalite/contentload/management/commands/unpack_assessment_zip.py]: I see. So the main purpose of the changes here is to show to the user that something is still happening.

**Comment by aronasorman** [comment]
Changes look good. Have you ever managed to rebase this @MCGallaspy?

**Comment by benjaoming** [comment]
I'll take this over, I'm doing a lot of changes in regards to this :)

**Pull request closed without merging**